proposals/Exceptions.md

```diff
@@ -128,9 +128,10 @@ Exception indices are used by:
 1. The `throw` instruction which creates a WebAssembly exception with the
    corresponding exception tag, and then throws it.
 
-2. The `if_except` instruction queries an exception to see if the corresponding
-   exception tag denoted by the exception index. If true it pushes the
-   corresponding values of the exception onto the stack.
+2. The `br_on_exn` instruction queries an exception to see if the corresponding
+   exception tag denoted by the exception index. If true it branches to the
+   given label and pushes the corresponding values of the exception onto the
```

Comment:
"value" -> "argument values" for clarity?

Reply:
Done